CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 1.0.0
+
+- Added support for ES6 module linting, which is activated if `"type": "module"` in your `package.json`
+- For Node versions 16 and above, the ES2022 environment will be activated
+- The test configuration has been changed, and should from now on be activated from the main `.eslintrc.json`-file,
+  see [README](./README.md#usage) for more information.
+
+### Migration Checklist
```

Comment:
Please make this migration optional, old projects already using the test config in a test/.eslintrc.json should not need to do anything

Reply:
I changed back the test configuration to work as before, so it is backwards compatible. The overrides now reside in the `all` config instead.